client/server/middleware/unsupported-browser.js

```diff
@@ -22,26 +32,25 @@ import analytics from 'calypso/server/lib/analytics';
  * listed in package.json. This list only serves as a way to assist users who are
  * using a browser which is definitely broken. It is not a guarantee that things
  * will work flawlessly on newer versions.
+ * @type {Map<string, (ua: UserAgentDetails) => boolean>}
  */
-const UNSUPPORTED_BROWSERS = [
-	'ie <= 11',
-	'edge <= 79',
-	'firefox <= 73',
-	'chrome <= 79',
-	'safari <= 13', // Note: 13.1 IS supported. Browserslist considers Safari point releases as new versions.
-	'opera <= 66',
-	'ios <= 13.3',
-];
+const UNSUPPORTED_BROWSERS = new Map( [
+	[ 'IE', () => true ],
+	[ 'Edge', ( ua ) => getMajorVersion( ua.version ) <= 79 ],
+	[ 'Firefox', ( ua ) => getMajorVersion( ua.version ) <= 73 ],
+	[ 'Chrome', ( ua ) => getMajorVersion( ua.version ) <= 79 ],
+	[ 'Safari', ( ua ) => getMajorVersion( ua.version ) <= 13 ],
+	[ 'Opera', ( ua ) => getMajorVersion( ua.version ) <= 66 ],
```

Comment:
That list looks seriously outdated. Officially, [we support only the last 2 major versions](https://github.com/automattic/wp-calypso/?tab=readme-ov-file#browser-support) of each browser, so I think a good follow-up PR would be to update that list to reflect that.

Reply:
As @jsnajdr mentioned in a private discussion, we might want to keep things as they are, since it reflects the list of browsers that are "surely very broken" (see existing code comment above). Could make sense to audit at some point, since we've been using newer and newer features without keeping an eye on what is supported and what isn't.